moveit_msgs/RobotTrajectory[] trajectory